app/src/main/java/org/mozilla/fenix/home/mozonline/PrivacyContentDisplayActivity.kt

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home.mozonline
+
+import android.app.Activity
+import android.content.Context
+import android.os.Bundle
+import android.util.AttributeSet
+import android.view.View
+import android.widget.ImageButton
+import mozilla.components.concept.engine.EngineSession
+import mozilla.components.concept.engine.EngineView
+import mozilla.components.feature.contextmenu.DefaultSelectionActionDelegate
+import mozilla.components.feature.search.BrowserStoreSearchAdapter
+import mozilla.components.support.ktx.android.content.call
+import mozilla.components.support.ktx.android.content.email
+import mozilla.components.support.ktx.android.content.share
+import org.mozilla.fenix.R
+import org.mozilla.fenix.ext.components
+
+/**
+ * A special activity for displaying the detail content about privacy hyperlinked in alert dialog.
+ */
+
+class PrivacyContentDisplayActivity : Activity(), EngineSession.Observer {
+    private var engineView: EngineView? = null
+    private var closeButton: ImageButton? = null
+    private var engineSession: EngineSession? = null
```

Comment:
nit: You could use [lateinit var](https://kotlinlang.org/docs/reference/properties.html#late-initialized-properties-and-variables) and avoid the null values :)